features/0212-pickup/README.md

```diff
@@ -70,7 +70,57 @@ A message that contains multiple waiting messages.
 {
     "@id": "123456781",
     "@type": "did:sov:BzCbsNYhMrjHiqZDTUASHg;spec/messagepickup/1.0/batch",
-    "messages": []
```

Comment:
How do you feel about `messages~attach` as the attribute/attachment method?

Reply:
+1 to @TelegramSam 's suggestion.